### PR TITLE
Cinder cleanup

### PR DESCRIPTION
**Pull request by toabctl**

### CODE REVIEW

**Comment by toabctl** [comment]
@Itxaka better?

**Comment by Itxaka** [comment]
@toabctl several options are still not changed regarding chap:

eqlx_chap_login
eqlx_chap_password
eqlx_use_chap

**Comment by toabctl** [comment]
@Itxaka thanks for thre hint! Hopefully fixed all now :)

File: chef/cookbooks/cinder/templates/default/cinder.conf.erb [hunk 1]
```diff
@@ -79,9 +88,9 @@ volume_driver = cinder.volume.drivers.emc.emc_vmax_iscsi.EMCVMAXISCSIDriver
   <% if volume['backend_driver'] == 'eqlx' -%>
 eqlx_group_name = <%= volume['eqlx']["eqlx_group_name"] %>
 eqlx_cli_timeout=<%= volume['eqlx']["eqlx_cli_timeout"] %>
-eqlx_use_chap = <%= volume['eqlx']["eqlx_use_chap"] %>
-eqlx_chap_login = <%= volume['eqlx']["eqlx_chap_login"] %>
-eqlx_chap_password = <%= volume['eqlx']["eqlx_chap_password"] %>
+use_chap_auth = <%= volume['eqlx']["eqlx_use_chap"] %>
```
Comment (Itxaka) [inline chef/cookbooks/cinder/templates/default/cinder.conf.erb]: `<%= volume['eqlx']["eqlx_use_chap"] %>` -> `<%= volume['eqlx']["use_chap_auth"] %>` and same below

Comment (toabctl) [inline chef/cookbooks/cinder/templates/default/cinder.conf.erb]: fixed

**Pull request merged**